perf-tests/README.md

```diff
@@ -0,0 +1,73 @@
+# seperate testing
+
+To start a server, run `perfTests/run` and select the server you want to test.
+
+To test vanilla akka-http load, run this command:
```

Comment:
vanilla is a good word! Maybe we can rename `AkkaHttpOnlyServer` to `AkkaHttpVanillaServer`? The "only" doesn't immediately convey what this implementation does :)